option optimize_for = LITE_RUNTIME;

message ChoukaRequest
{
 required uint32 isdef = 1;
 required uint32 act_id = 2;
 required uint32 cfg_id = 3;
 required uint32 count = 4;
}
